jobserver/views/projects.py

```diff
@@ -131,22 +131,43 @@ def post(self, request, *args, **kwargs):
 
 
 @method_decorator(require_superuser, name="dispatch")
-class ProjectSettings(TemplateView):
+class ProjectSettings(CreateView):
+    form_class = ProjectInvitationForm
+    model = ProjectInvitation
     template_name = "project_settings.html"
 
-    def get_context_data(self, **kwargs):
-        project = get_object_or_404(
+    def dispatch(self, request, *args, **kwargs):
+        self.project = get_object_or_404(
             Project,
             org__slug=self.kwargs["org_slug"],
             slug=self.kwargs["project_slug"],
         )
 
-        can_manage_members = has_permission(
+        self.can_manage_members = has_permission(
             self.request.user,
             "manage_project_members",
-            project=project,
+            project=self.project,
+        )
+        return super().dispatch(request, *args, **kwargs)
+
+    def form_valid(self, form):
```

Comment:
Are we missing a check here that the user has permission to create the invitations?

Reply:
Yes… 😬